I ran into these issues trying to build the patch:

```
[mareynol@fedora 389-console]$ ./buildAndRun.sh 

Cleaning and installing npm packages ...


Building and watching ...

  src/css/ds.css:428:19
  ✖  428:19  Expected context media feature range notation  media-feature-range-notation
  ✖  456:19  Expected context media feature range notation  media-feature-range-notation

  2 errors

14:41:20: Build finished in 6271 ms
✘ [ERROR] stylelint errors found [plugin stylelintPlugin]
```

@droideck can you apply this diff to your patch to fix these style errors:

```
[mareynol@fedora 389-ds-base]$ git diff
diff --git a/src/cockpit/389-console/src/css/ds.css b/src/cockpit/389-console/src/css/ds.css
index f1abbdc17..9cd669569 100644
--- a/src/cockpit/389-console/src/css/ds.css
+++ b/src/cockpit/389-console/src/css/ds.css
@@ -425,7 +425,7 @@ textarea {
     margin-left: 80px;
 }
 
-@media screen and (max-width: 1300px) {
+@media screen and (max-width >= 1300px) {
     .ds-plugin-spinner {
         margin-left: 0;
     }
@@ -453,7 +453,7 @@ textarea {
     top: -16px;
 }
 
-@media screen and (max-width: 770px) {
+@media screen and (max-width >= 770px) {
     .ds-plugin-tab-header {
         margin-top: 0;
         margin-bottom: 0;
```

--- a/src/cockpit/389-console/src/css/ds.css
+++ b/src/cockpit/389-console/src/css/ds.css
@@ -425,7 +425,7 @@ textarea {
     margin-left: 80px;
 }
 
-@media screen and (max-width: 1300px) {
+@media screen and (max-width >= 1300px) {
     .ds-plugin-spinner {
         margin-left: 0;
     }
@@ -453,7 +453,7 @@ textarea {
     top: -16px;
 }
 
-@media screen and (max-width: 770px) {
+@media screen and (max-width >= 770px) {
     .ds-plugin-tab-header {
         margin-top: 0;
         margin-bottom: 0;

